.github/workflows/pr.yaml

```diff
@@ -79,6 +67,17 @@ jobs:
           do
             sleep 3
           done
+      - name: Deploy/Test APIServer embedded KONK
```

Comment:
What's the difference between this and the previous test?

Reply:
embedded KONK vs non-embedded KONK. I don't really have a way to cheat at this. Perhaps we deploy both examples at the same time 🤔

Reply:
In parallel independent workflows? SGTM.

Reply:
All the example-apiserver tests should probably move to a separate workflow. I left out the embedded test for now.